modules/higherKindCore/src/test/scala/tofu/control/CatsInstancesFromBindSuite.scala

```diff
@@ -0,0 +1,16 @@
+package tofu.control
+
+import cats.FlatMap
+import tofu.compat.unused
+import tofu.instances.bind._
+
+class CatsInstancesFromBindSuite {
+  def summonBindInstances[F[+_, +_]](implicit bind: Bind[F]): Unit = {
+    requireApplicative[F[Throwable, *]]
+    requireFlatMap[F[Nothing, *]]
+  }
+
+  def requireApplicative[F[_]](implicit @unused applicative: FlatMap[F]): Unit = ()
```

Comment:
Actually you summon FlatMap, not an Applicative

Reply:
Thank you, fixed